app/assets/javascripts/angularjs-nvd3-directives/angularjs-nvd3-directives.min.js

```diff
@@ -0,0 +1,3 @@
owYAxis(void 0===attrs.showyaxis?!1:"true"===attrs.showyaxis).tooltips(void 0===attrs.tooltips?!1:"true"===attrs.tooltips).noData(void 0===attrs.nodata?"No Data Available.":scope.nodata).interactive(void 0===attrs.interactive?!1:"true"===attrs.interactive).clipEdge(void 0===attrs.clipedge?!1:"true"===attrs.clipedge).color(void 0===attrs.color?nv.utils.defaultColor():scope.color());return chart.useInteractiveGuideline&&chart.useInteractiveGuideline(void 0===attrs.useinteractiveguideline?!1:"true"===attrs.useinteractiveguideline),attrs.usevoronoi&&chart.useVoronoi("true"===attrs.usevoronoi),attrs.style&&chart.style(attrs.style),attrs.order&&chart.order(attrs.order),attrs.offset&&chart.offset(attrs.offset),attrs.interpolate&&chart.interpolate(attrs.interpolate),attrs.tooltipcontent&&chart.tooltipContent(scope.tooltipcontent()),attrs.xscale&&chart.xScale(scope.xscale()),attrs.yscale&&chart.yScale(scope.yscale()),attrs.xdomain&&(Array.isArray(scope.$eval(attrs.xdomain))?chart.xDomain(scope.$eval(attrs.xdomain)):"function"==typeof scope.xdomain()&&chart.xDomain(scope.xdomain())),attrs.ydomain&&(Array.isArray(scope.$eval(attrs.ydomain))?chart.yDomain(scope.$eval(attrs.ydomain)):"function"==typeof scope.ydomain()&&chart.yDomain(scope.ydomain())),attrs.sizedomain&&chart.sizeDomain(scope.sizedomain()),scope.d3Call(data,chart),nv.utils.windowResize(chart.update),scope.chart=chart,chart},callback:void 0===attrs.callback?null:scope.callback()})}},void 0===attrs.objectequality?!1:"true"===attrs.objectequality)}}}]).directive("nvd3MultiBarChart",[function(){return{restrict:"EA",scope:{data:"=",width:"@",height:"@",id:"@",showlegend:"@",tooltips:"@",tooltipcontent:"&",color:"&",showcontrols:"@",nodata:"@",reducexticks:"@",staggerlabels:"@",rotatelabels:"@",margin:"&",x:"&",y:"&",forcey:"@",delay:"@",stacked:"@",callback:"&",showxaxis:"&",xaxisorient:"&",xaxisticks:"&",xaxistickvalues:"&xaxistickvalues",xaxisticksubdivide:"&",xaxisticksize:"&",xaxistickpadding:"&",xaxistickformat:"&",xaxislabel:"@",xaxisscale:"&",xaxisdomain:"&",xaxisrange:"&",xaxisrangeband:"&",xaxisrangebands:"&",xaxisshowmaxmin:"@",xaxishighlightzero:"@",xaxisrotatelabels:"@",xaxisrotateylabel:"@",xaxisstaggerlabels:"@",xaxisaxislabeldistance:"@",showyaxis:"&",yaxisorient:"&",yaxisticks:"&",yaxistickvalues:"&yaxistickvalues",yaxisticksubdivide:"&",yaxisticksize:"&",yaxistickpadding:"&",yaxistickformat:"&",yaxislabel:"@",yaxisscale:"&",yaxisdomain:"&",yaxisrange:"&",yaxisrangeband:"&",yaxisrangebands:"&",yaxisshowmaxmin:"@",yaxishighlightzero:"@",yaxisrotatelabels:"@",yaxisrotateylabel:"@",yaxisstaggerlabels:"@",yaxislabeldistance:"@",legendmargin:"&",legendwidth:"@",legendheight:"@",legendkey:"@",legendcolor:"&",legendalign:"@",legendrightalign:"@",legendupdatestate:"@",legendradiobuttonmode:"@",objectequality:"@",transitionduration:"@"},controller:["$scope","$element","$attrs",function($scope,$element,$attrs){$scope.d3Call=function(data,chart){checkElementID($scope,$attrs,$element,chart,data)
```

Comment:
Line is too long.<br>Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.<br>Too many errors. (25% scanned).